data/templates/fail2ban/jail.conf

```diff
@@ -581,4 +581,5 @@ enabled  = true
 port     = http,https
 protocol = tcp
 filter   = yunohost
-logpath  = /var/log/nginx/*.log
+logpath  = /var/log/nginx*/*error.log
```

Comment:
nope, it's probably not needed; It's only copy/pasta from others nginx sections in jail.conf (https://github.com/YunoHost/yunohost/blob/unstable/data/templates/fail2ban/jail.conf#L554-L576)